packages/edit-site/src/components/page-patterns/use-patterns.js

```diff
@@ -93,11 +93,18 @@ const selectThemePatterns = ( select, { categoryId, search = '' } = {} ) => {
 			blocks: parse( pattern.content ),
 		} ) );
 
-	patterns = searchItems( patterns, search, {
-		categoryId,
-		hasCategory: ( item, currentCategory ) =>
-			item.categories?.includes( currentCategory ),
-	} );
+	if ( categoryId ) {
+		patterns = searchItems( patterns, search, {
+			categoryId,
+			hasCategory: ( item, currentCategory ) =>
+				item.categories?.includes( currentCategory ),
+		} );
+	} else {
+		// If there is no category id, select uncategorized patterns.
+		patterns = patterns.filter(
+			( item ) => ! item.hasOwnProperty( 'categories' )
```

Comment:
The uncategorized patterns would still need to be filtered via the search term correct?

Reply:
Yes you are right, let me see if I can fix that 🤔

Reply:
I still don't understand why the search was limited to searching inside categories instead of across all.